server/controllers/v1/tests/post.test.ts

```diff
@@ -71,6 +74,86 @@ describe('Test v1 - Post Controller', () => {
       expect(result.status).toBe(200);
     });
 
+    it('should allow creating new tags when making a post', async () => {
+      const author = await makeValidUser();
+      const tags = ['csc108', 'bananaPepper'];
+
+      const result = await postController.createPost(
+        author.id,
+        'This is a new post!',
+        'This is a new post!This is a new post!',
+        'location',
+        10,
+        tags
+      );
+
+      if (result.data.result) {
+        const postTags = await getPostTags(result.data.result.id);
+        expect(postTags?.map((t) => t.text).sort()).toEqual(tags.sort());
+      } else {
+        expect(true).toBe(false);
+      }
+
+      expect(result.status).toBe(200);
+    });
+
+    it('should be able to use existing tags', async () => {
```

Comment:
What's this testing? That tags aren't re-created? If so, that's more of a model test.

Reply:
It's specifically testing if the controller is properly handling duplicaet tags

Reply:
It's not the controller that handles duplicates though. I would move it to the tag model test

Reply:
alright